.env.local.example

```diff
@@ -32,7 +32,7 @@ GOOGLE_SECRET=
 
 ## Stripe ##
 ##############
-STRIPE_PUBLIC_KEY=
+STRIPE_PUBLISHABLE_KEY=
```

Comment:
We may need to adapt the k8s manifests to adopt this change in the env var name.